apache/cloudstack
# VMware: match hardware version for worker VM when taking a snapshot

## Description

When a worker VM is created in the process of creating a volume snapshot on Vmware, the hardware version of the worker VM will be set to the highest supported hardware version of the hypervisor. If you have hypervisors of different versions in a cluster and have set the "Default VM compatibility" to a lower version, the worker VM should use the same version as the original VM instance,

This PR sets the worker VM hardware version to the same version as the original VM instance.








## Types of changes


## Screenshots (if appropriate):

## How Has This Been Tested?



This has been tested by doing the following: 
* create an instance 
* take a snapshot 
* check the hardware version of the worker vm created in Vsphere.
